### PR TITLE
Adding roundness documentation

### PR DESCRIPTION
**Pull request by desig9stein**

### CODE REVIEW

File: en/components/roundness.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+If you decide to go with a unitless value from 0 to 1, you will automatically be limited by the theming engine to min and max boundary for each component.
+If you pass any kind of unit:, px, em, rem, %... you will no longer be restricted to the boundaries for the component. 
+So for example, if you set a value of 30px the theming engine will ignore the boundaries of maximum 20px and will set the border-radius to 30px.
+
+### How to use?
+Let see how we can change the default values for those buttons from the above example.
+
+```scss
+// In the context of button the min and max values are 0 - 20px,
```
Comment (simeonoff) [inline en/components/roundness.md]: In the context of the button

File: en/components/roundness.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+If you decide to go with a unitless value from 0 to 1, you will automatically be limited by the theming engine to min and max boundary for each component.
+If you pass any kind of unit:, px, em, rem, %... you will no longer be restricted to the boundaries for the component. 
+So for example, if you set a value of 30px the theming engine will ignore the boundaries of maximum 20px and will set the border-radius to 30px.
+
+### How to use?
+Let see how we can change the default values for those buttons from the above example.
+
+```scss
+// In the context of button the min and max values are 0 - 20px, 
+// For all baseline values see the table on the bottom of the page
```
Comment (simeonoff) [inline en/components/roundness.md]: see the table *__at__*

File: en/components/roundness.md [hunk 3]
```diff
@@ -26,18 +26,18 @@ As you can see from the example above, the component schema for [Button theme]({
 Let's look at how things work. 
 The default value for "flat-border-radius" is set to 0.2 which in the end will be resolved to 4px, it is actually a fraction between 0 and 20px where 0 is the minimum border-radius and 20px is the maximum. 
 
-We decided not to limit You to set just a fraction between the min and max values, but to give you an option to use whatever value you want. 
+We decided to not limit you to fractions only. You can use whatever unit you want - pixels, relative units like em or rem, etc., allowing you to overwrite the implicit border radius limits.
 
-If you decide to go with a unitless value from 0 to 1, you will automatically be limited by the theming engine to min and max boundary for each component.
-If you pass any kind of unit:, px, em, rem, %... you will no longer be restricted to the boundaries for the component. 
-So for example, if you set a value of 30px the theming engine will ignore the boundaries of maximum 20px and will set the border-radius to 30px.
+If you decide to go with a fractions, you will automatically be limited by the theming engine to min and max boundary for each component.
```
Comment (simeonoff) [inline en/components/roundness.md]: If you decide to go with fractions

**Comment by StefanIvanov** [comment]
@desig9stein can you fix the conflicts in the TOC?

**Pull request merged**